README.md

```diff
@@ -125,10 +126,33 @@ Mitigating this threat is achieved by:
 ]
 ```
 
+## Safe Keyboard Detector
+
+⚠ Warning : This functionality is ONLY implemented on Android since iOS already prevent the use of third party keyboard on sensitive filed such as passwords. 
```

Comment:
```suggestion
⚠ Warning : This functionality is ONLY implemented on Android since iOS already prevents the use of third party keyboards on sensitive fields such as passwords. 
```